.github/workflows/release.yml

```diff
@@ -1,137 +1,104 @@
-# CI that:
```

Comment:
I may need to do some fiddling with this after merge since we can't really test this.
If so I'll temporarily disable branch protection and just mess around in `main` until it's working.